README.md

```diff
@@ -13,6 +13,7 @@ For support questions around this collection, please open a ticket on the [Red H
 ## Installation
 
 You can install this collection directly from Automation Hub or via RPMs provided by Red Hat.
+In both cases you will require the `python2-apypie` RPM provided in the Red Hat Satellite repositories.
```

Comment:
given the collection rpm has a dependency on that, maybe adding it to the AH section is nicer?

Reply:
Reads good... should we drop the `or via RPMs...` in the line above for now to reflect real current state?

Reply:
dropped all "via rpm" stuff as there are no rpms (yet) and moved that sentence to AH